requirements/pytorch/loggers.info

```diff
@@ -1,5 +1,5 @@
 # all supported loggers. this list is here as a reference, but they are not installed in CI
 neptune-client
 comet-ml
-mlflow
+mlflow>=1
```

Comment:
@Borda I couldn't find where we are specifying the minimum version for loggers. Let's add it here?